src/js/pages/system/UnitsHealthDetail.js

```diff
@@ -0,0 +1,144 @@
+import mixin from 'reactjs-mixin';
+/* eslint-disable no-unused-vars */
+import React from 'react';
+/* eslint-enable no-unused-vars */
+import {StoreMixin} from 'mesosphere-shared-reactjs';
+
+import FilterBar from '../../components/FilterBar';
+import FilterHeadline from '../../components/FilterHeadline';
+import FilterInputText from '../../components/FilterInputText';
+import Header from '../../components/Header';
+import RequestErrorMsg from '../../components/RequestErrorMsg';
+import UnitHealthDropdown from '../../components/UnitHealthDropdown';
+import UnitHealthNodesTable from '../../components/UnitHealthNodesTable';
+import UnitHealthStore from '../../stores/UnitHealthStore';
+
+const METHODS_TO_BIND = [
+  'handleHealthSelection',
+  'handleSearchStringChange',
+  'resetFilter'
+];
+
+class UnitsHealthDetail extends mixin(StoreMixin) {
+  constructor() {
+    super(...arguments);
+
+    this.store_listeners = [
+      {
+        name: 'unitHealth',
+        events: ['unitSuccess', 'unitError', 'nodesSuccess', 'nodesError']
+      }
+    ];
+
+    this.state = {
+      healthFilter: 'all',
+      searchString: ''
+    };
+
+    METHODS_TO_BIND.forEach((method) => {
+      this[method] = this[method].bind(this);
+    }, this);
```

Comment:
Setting the context explicitly with `this`  is not needed here, because you are using a fat-arrow function. That is done automatically.